modules/videopress/class.videopress-gutenberg.php

```diff
@@ -6,8 +6,16 @@ class VideoPress_Gutenberg {
 	 * Initialize the VideoPress Gutenberg extension
 	 */
 	public static function init() {
-		// Should not initialize if Gutenberg is not available
-		if ( function_exists( 'register_block_type' ) ) {
+		// Should not initialize if Gutenberg is not available or if Jetpack is not active
+		if (
+			( function_exists( 'register_block_type' ) )
+			&&
+			(
+				( method_exists( 'Jetpack', 'is_active' ) && Jetpack::is_active() )
+				||
```

Comment:
Have you got `phpcs` running? The formatting in these blocks is off, but my plugin seems borked, so I'm having a hard time telling what it should be. Running from the CLI gives me this:

<img width="707" alt="screen shot 2019-02-05 at 8 52 51 am" src="https://user-images.githubusercontent.com/349751/52289830-a330f680-2923-11e9-81da-b356f2b44707.png">

Reply:
Those are mostly minor notices so it's not too bad. I committed be32b69c6af3266ad73982788a619d81668de9ff to clean this up a bit.

Reply:
Thanks @kwight, @jeherve! Indeed, I didn't have running `phpcs`, so I'll make sure to set it up for the next time.

Reply:
@mmtr it's a beast, lemme know if I can help :)